webplugin/pluginOptimizer.js

```diff
@@ -124,22 +124,24 @@ const compressAndOptimize = () => {
                 drop_console: true
             }
         },
-        output: path.resolve(__dirname, `${buildDir}/kommunicate-plugin.${version}.min.js`),
+        output: path.resolve(__dirname, `${buildDir}/kommunicate-plugin.min.js`),
```

Comment:
@ab14bhardwaj is ${version} required here? if not, will it handle the browser cache issue?

Reply:
@devashishmamgain It is not required here because the above file is not getting served anywhere.